### PR TITLE
rpc: Make rpc documentation not depend on call-time rpc args

### PR DESCRIPTION
**Pull request by maflcko**
This is required to host the documentation on a static resource (like a website or pdf)

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 
 static const std::string WALLET_ENDPOINT_BASE = "/wallet/";
+static const std::string HELP_REQUIRING_PASSPHRASE{"\nRequires wallet passphrase to be set with walletpassphrase call if wallet is encrypted.\n"};
```
Comment (emilengler) [inline src/wallet/rpcwallet.cpp:41]: nit: The line above is not using a C++11 list init. I would change it to a C one to keep it coherent.

**Review by cvengler — COMMENTED** [review]
Concept ACK, a static documentation is most likely a better choice

**Review by theStack — COMMENTED** [review]
Concept ACK

**Review by promag — COMMENTED** [review]
ACK fab32557f25c93fecb7bb33f78fdf3e9160c89e5.

**Comment by emilengler** [comment]
utACK fab32557f25c93fecb7bb33f78fdf3e9160c89e5

**Pull request merged**